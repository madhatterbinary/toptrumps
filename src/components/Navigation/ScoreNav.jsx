import React from 'react';
import PropTypes from 'prop-types';

const ScoreNav = ({ player1Score, player2Score }) => (
  <div className="score-nav">
    <div className="player-scores">
      <div><span className="player-score">{player1Score}</span></div>
      <div><span className="player-score">{player2Score}</span></div>
    </div>
  </div>
);

ScoreNav.propTypes = {
  player1Score: PropTypes.number,
  player2Score: PropTypes.number
};

export default ScoreNav;
