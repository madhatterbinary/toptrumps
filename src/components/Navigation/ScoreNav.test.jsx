import React from 'react';
import ScoreNav from './ScoreNav';
import TestRenderer from 'react-test-renderer';

describe('ScoreNav', () => {
  it('renders', () => {
    const component = TestRenderer.create(<ScoreNav />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
