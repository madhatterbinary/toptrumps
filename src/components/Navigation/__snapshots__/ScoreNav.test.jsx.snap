// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScoreNav renders 1`] = `
<div
  className="score-nav"
>
  <div
    className="player-scores"
  >
    <div>
      <span
        className="player-score"
      />
    </div>
    <div>
      <span
        className="player-score"
      />
    </div>
  </div>
</div>
`;
